responses/08_workflows-answer.md

```diff
@@ -8,7 +8,12 @@ This pull request covers the basics, but here are some recommendations to help a
 - If you haven't picked a workflow, use a common and well documented one.
 - Be explicit about your deployment and release strategy.
 
-Please merge this pull request.
+### :keyboard: Activity: Review workflow documentation
+
+1. Review the changes in the `Files changed` tab
+2. Approve this pull request (if needed)
```

Comment:
@brianamarie why would it be optional? Seems like if the bot is successful at removing the branch protection, the learner actually only needs to merge. If the bot isn't successful, why not? I'm not sure how to reproduce #26, do you have a little more information?

Reply:
You're right - _if_ the bot is successful at removing the branch protection. This was happening inconsistently for me during the course. Unless we're having the time to fix the config bug before the weekend, I was hoping we could merge this so users have better instructions to get through the course, until we can get to fixing the config.

Reply:
I've not received that error, but I often have to manually refresh to see that the protection has actually been removed. 

I say we:
1. Include your ✨ approval instruction.
2. Make the remove branch protection step optional (meaning that if LL tries to remove the branch protection and can't, say because the learner has manually removed it), then the step won't fail.

Reply:
Correction to ☝️, that action is [already optional](https://github.com/githubtraining/innersource/blob/master/config.yml#L136-L137). Even though you saw the error, did the course actually stop working, @brianamarie ?

Reply:
@hectorsector nope, it let me finish. The main problem, aside from the debugging issue that got opened (do those open for non-admins?), was the confusion in the instructions when I was being asked to merge but couldn't due to branch protections.